pytorch_lightning/utilities/cli.py

```diff
@@ -160,11 +160,13 @@ def __init__(
         config: Union[Namespace, Dict[str, Any]],
         config_filename: str,
         overwrite: bool = False,
+        multifile: bool = False,
```

Comment:
becoming quite complex, pls add full args docstring

Reply:
This is in the CLI docstring